.github/workflows/deploy-alpha.yml

```diff
@@ -30,7 +30,7 @@ jobs:
           git checkout .
       - name: Build packages used by others
         run: |
-          npm run build-essential
+          npm run build:essential
```

Comment:
If this is applied to master, all the other branches that are not synced will not be ablet odeploy anymore